book/glossary/glossary.tex

```diff
@@ -0,0 +1,374 @@
+
+1: \textit{Yeast Activity \& Oven Spring:} In the initial phase of baking, the
+temperature inside the dough rises, increasing yeast activity. This results in rapid
+carbon dioxide production, leading to what bakers refer to as ``oven spring'', or the
+rapid rise of the loaf.
+
+2: \textit{Protein Coagulation:} As the temperature continues to climb, the proteins
+in the dough, primarily gluten, begin to coagulate or set, which gives the bread its
+structure.
+
+3: \textit{Starch Gelatinization:} Starches absorb water and swell, eventually
+gelatinizing. This process contributes to the crumb structure of the bread.
+
+4: \textit{Caramelization \& Maillard Reaction:} The crust of the bread browns due
+to two primary reactions: caramelization of sugars and the Maillard reaction between
+amino acids and reducing sugars. This not only affects the appearance but also imparts
+a distinctive flavor and aroma to the bread.
+
+5: \textit{Evaporation of Acids:} Some acids produced during fermentation evaporate at
+certain temperatures during baking. This evaporation can influence the final flavor
+profile of the bread, making it less tangy than the unbaked dough. By extending the
+baking time the acids become less concentrated and the dough can lose some of its tang.
+
+6: \textit{Moisture Evaporation:} Water in the dough turns to steam and begins to
+evaporate. The steam contributes to the oven spring and also helps in gelatinizing
+the starches.
+
+7: \textit{Crust Formation:} The outer layer of the dough dries out and hardens to
+form a crust, which acts as a protective barrier, keeping the inner crumb moist.
+
+\textbf{Banneton:} A wicker basket used to shape and support dough during its final
+proof. The bannetons are typically made out of rattan or wood pulp. An alternative
+DIY solution is to use a bowl with a kitchen towel inside. While resting inside of
+the banneton the dough’s surface dries out and becomes easier to score before baking.
+
+\textbf{Bench rest:} A short resting period given to the dough after preshaping
+allowing the gluten to relax a little bit and making shaping easier. Most people
+bench rest for 10 minutes up to an hour. The bench rest becomes especially important
+when making pizza doughs. Without an extended bench rest the dough is too elastic and
+can not be shaped.
+
+\textbf{Beta-amylase:} An enzyme that further breaks down the starch fragments
+produced by alpha-amylase into maltose.
+
+\textbf{Bread flour:} A flour that is perfect for sourdough bread making. It features
+a higher amount of gluten and can thus ferment for a longer period of time.
+
+\textbf{Brühstück:} A German baking technique similar to a scald. It translates as
+``boil piece''. Hot or boiling water is poured over whole grain flour or crushed grains,
+then cooled and mixed with the main dough. This process helps in moisture retention
+and can enhance the flavor and texture of the final bread. Also see \#Scald.
```

Comment:
Is \# creating an hyperlink of some sort ?

Reply:
Yes, just a reference. No need for an actual hyperlink I think.